lattice/lattice.py

```diff
@@ -156,7 +158,7 @@ def setup_json_schemas(self):
   def generate_json_schemas(self):
     generate_core_json_schema(os.path.join(self.json_schema_directory, "core.schema.json"))
```

Comment:
Shouldn't the core schema be handled in memory too, since it will only be used as a reference in the other schemas and flattened out?